tools/wptserve/setup.py

```diff
@@ -1,9 +1,13 @@
 from setuptools import setup
 
+import os
+
+here = os.path.dirname(__file__)
+
 PACKAGE_VERSION = '4.0.1'
```

Comment:
@jgraham I assume that you will bump and release in a follow-up PR or can we combine the release here as well?